### PR TITLE
configuring scheduler name from configmap

### PR DESCRIPTION
**Pull request by itayvallach**
Set environment variable SCHEDULER_NAME in"notebook-controller-parameters" ConfigMap and it will be used as the notebook scheduler.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/4525)

### CODE REVIEW

**Comment by itayvallach** [comment]
@googlebot I signed it!

File: components/notebook-controller/controllers/notebook_controller.go [hunk 1]
```diff
@@ -328,6 +328,10 @@ func generateStatefulSet(instance *v1beta1.Notebook) *appsv1.StatefulSet {
 			FSGroup: &fsGroup,
 		}
 	}
+	schedulerName := os.Getenv("SCHEDULER_NAME")
+	if len(schedulerName) > 0 {
+		podSpec.SchedulerName = schedulerName
+	}
```
Comment (yeya24) [inline components/notebook-controller/controllers/notebook_controller.go]: Don' t think this is a good way to configure the scheduler in notebook controller. We should control this through the notebook CR itself

**Pull request closed without merging**